crates/holochain/src/conductor/conductor.rs

```diff
@@ -1883,9 +1913,7 @@ mod clone_cell_impls {
     impl Conductor {
         /// Create a new cell in an existing app based on an existing DNA.
         ///
-        /// # Returns
-        ///
-        /// A struct with the created cell's clone id and cell id.
```

Comment:
This was self-evident and is discouraged https://doc.rust-lang.org/rustdoc/how-to-write-documentation.html#documenting-components